jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/format/TraceFormat.java

```diff
@@ -86,7 +86,7 @@
 
 	// constants
 	protected static final String usageMessage = "Usage:\n"
-			+ "java com.ibm.jvm.format.TraceFormat input_filespec [output_filespec] \n"
+			+ "traceformat input_filespec [output_filespec] \n"
```

Comment:
We still need to support IBM builds of Java 8 which don't have the traceformat executable. This should use the preprocessor to select which output to use.